src/main/kotlin/io/sip3/captain/ce/pipeline/RtpHandler.kt

```diff
@@ -33,14 +34,29 @@ class RtpHandler(context: Context, bulkOperationsEnabled: Boolean) : Handler(con
 
     private val packets = mutableListOf<Packet>()
     private var bulkSize = 1
+    private var payloadTypes: Set<Byte> = emptySet()
 
     private val vertx: Vertx
 
     init {
-        if (bulkOperationsEnabled) {
-            context.config().getJsonObject("rtp")?.let { config ->
+        context.config().getJsonObject("rtp")?.let { config ->
+            if (bulkOperationsEnabled) {
                 config.getInteger("bulk-size")?.let { bulkSize = it }
             }
+
+            config.getJsonArray("payload-types")?.let { array ->
+                val result = mutableSetOf<Byte>()
```

Comment:
Can't we just do a `flatMap` instead of using `result`? Also with `flatMap` we can get rid of `array` as a parameter name it's not very descriptive IMO.